spec/lib/rex/post/meterpreter/packet_spec.rb

```diff
@@ -147,7 +147,7 @@
       end
 
       it "should show the correct type and meta type in inspect" do
-        tlv_to_s = "#<Rex::Post::Meterpreter::Tlv type=COMMAND-ID      meta=INT        value=31337 command=unknown>"
+        tlv_to_s = "#<Rex::Post::Meterpreter::Tlv type=COMMAND_ID      meta=INT        value=31337 command=unknown>"
```

Comment:
Could we throw in a test for verifying `oneOf` and unknown scenarios, just so if someone changes the API in the future - some tests break